js/views/pageNavVw.js

```diff
@@ -479,6 +480,14 @@ module.exports = Backbone.View.extend({
     this.currentWindow.reload();
   },
 
+  addressBarFocus: function(e){
+    // on inital focus of input, select all text (this makes it easier to copy or delete the text)
+    $(this.$el).one('mouseup', function () {
+      $('#addressBar').select()
+      return false;
+    }).select();
```

Comment:
See http://stackoverflow.com/questions/3150275/jquery-input-select-all-on-focus/18534059#18534059

This approach highlights all the text on the initial click and then any subsequent click it does not highlight the text. This is important because it allows the user to use the input normally after the initial click.